client/src/Register.css

```diff
@@ -0,0 +1,69 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+
+.register {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;    
+    height: 100vh; 
+}
+
+.register_header {
+    color: #00849c;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 48px;
+    margin-bottom: 20px;
+    margin-top: 10px;
+}
+
+.textInputSet {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start; 
+}
+
+.inputType {
+    color: #81a1b7;
+    font-family: 'Montserrat', sans-serif; 
+    font-size: 18px; 
+    font-weight: 700; 
+    margin-bottom: 0px;
+}
+
+.accountButton {
+    width: 360px;
+    height: 56px;
+    background-color: #00849c;
+    font-size: 18px;
+    font-weight: bold;
+    font-family: 'Montserrat', sans-serif;
+    color: white;
+    border: none;
+    border-radius: 9px;
+    outline: none;
+}
+
+.inputBox {
+    border-color: #81A1B7;
+    border-radius: 9px;
+    border-width: 1px;
+    border-style: solid;
+    width: 360px;
```

Comment:
Let's take out this width so this page looks good on mobile. It currently looks like this on iPhoneX.

<img width="410" alt="Screen Shot 2020-10-16 at 11 19 43 PM" src="https://user-images.githubusercontent.com/20432204/96330011-dd163380-1006-11eb-88bd-e795b58d3042.png">